### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
### Capstone project RWD 😎

Developet with:

- HTML
- CSS
- JavaScript

Added:

1.  Responsibe design.
2. Mobile and desktop menu.
3. Live version.

This is my [video](https://www.loom.com/share/0ea8113063bf40e4995f24516481e93a) describing my project. ☺

### CODE REVIEW

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```
Comment (Emmyn5600) [inline .hintrc]: - [x] Nice to have linters display green as needed however, it seems like you didn't follow the Microverse [template](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json) to setup the linters, please go back and set up again using the template provided.

File: js/about.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const menu = document.querySelector('.open');
 menu.addEventListener('click', () => {document.getElementById('mobile-menu').classList.add('active') })
 const close = document.querySelector('.close');
-close.addEventListener('click', () => {document.getElementById('mobile-menu').classList.remove('active')})
\ No newline at end of file
+close.addEventListener('click', () => {document.getElementById('mobile-menu').classList.remove('active')})
+const ulClose = document.querySelector('.ul-close');
+ulClose.addEventListener('click', () => {document.getElementById('mobile-menu').classList.remove('active')})
```
Comment (Emmyn5600) [inline js/about.js]: - [x] The mobile size contains an hamburger menu on the left side, nice you have it, however, the hamburger menu should disappear once a user clicks on it and appear the close button on the right side.

File: index.html [hunk 3]
```diff
@@ -26,10 +26,10 @@
                 </section>
                 <div id="mobile-menu" class="list-mobile">
                     <span class="close">x</span>
-                    <ul>
+                    <ul class="ul-close">
                         <li><a href="/about.html">About</a></li>
-                        <li><a href="#"></a>Biography</li>
-                        <li><a href="#"></a>Projects</li>
+                        <li><a href="#champions">Facts</a></li>
+                        <li><a href="#architect">Speakers</a></li>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Your about file seems to be broken. take a quick look through to make it work, check the screenshot below:
about1

File: index.html [hunk 4]
```diff
@@ -38,15 +38,14 @@
                 <p class="logo">Architects</p>
                 <ul class="nav-list flex">
                     <li><a href="/about.html">About</a></li>
-                    <li><a href="">Prizes</a></li>
-                    <li><a href="">Biography</a></li>
-                    <li><a href="">Projects</a></li>
+                    <li><a href="#champions">Facts</a></li>
+                    <li><a href="#architect">Speakers</a></li>
```
Comment (Emmyn5600) [inline index.html:49]: - [ ] Please follow the design provided to implement some features, Compare the two images:

--------------------------------------Original-----------------------------------------------
nav1

--------------------------------------Yoursl-----------------------------------------------
nav2

File: index.html [hunk 5]
```diff
@@ -72,45 +71,46 @@ <h2>Facts</h2>
...
 
                 <div class="more_info flex">
-                    <a href="#"> See more information</a>
+                    <a href="https://www.pritzkerprize.com/"> See more information</a>
                 </div>
             </div>
             </section>
+            <h2 class="featured">Featured Speakers</h2>
 
             <section id="architect" class="grid_Winer">
```
Comment (Emmyn5600) [inline index.html]: - [ ] As stated above,  Please follow the design provided to implement some features, Compare the two images again:

--------------------------------------Original-----------------------------------------------
nav3

--------------------------------------Yours-----------------------------------------------
nav4

File: index.html [hunk 6]
```diff
@@ -120,9 +120,9 @@ <h3>Best Design</h3>
                    <p>Partner</p>
                 </div>
                 <div class="flex img_footer">
-                    <img src="/media/hamza.jpeg" width="150px" alt="">
-                    <img src="/media/google.jpg" width="150px" alt="">
-                    <img src="/media/github.jpg" width="150px" alt="">
+                    <img src="/media/hamza.jpeg" width="150" alt="">
+                    <img src="/media/google.jpg" width="150" alt="">
+                    <img src="/media/github.jpg" width="150" alt="">
```
Comment (Emmyn5600) [inline index.html]: - [ ] As stated above,  Please follow the design provided to implement some features, Compare the two images again:

--------------------------------------Original-----------------------------------------------
nav6

--------------------------------------Yours-----------------------------------------------
nav5

File: css/about.css [hunk 7]
```diff
@@ -126,101 +126,103 @@ hr {
...
+    justify-content: flex-end;
+  }
+
+  .section-social {
+    margin: 20px 8px 8px;
+    width: 20%;
+    justify-content: space-between;
+  }
+
+  .imgs {
```
Comment (Emmyn5600) [inline css/about.css]: - [ ] As stated above,  Please follow the design provided to implement some features, Compare the two images again:

--------------------------------------Original-----------------------------------------------
nav7



--------------------------------------Yours-----------------------------------------------
nav9

File: index.html [hunk 8]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="https://use.fontawesome.com/741f09be9f.js"></script>
     <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
     <link rel="stylesheet" href="./css/styles.css">
```
Comment (Emmyn5600) [inline index.html]: - [x] Let us change our title of the project with the descriptive name.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @xJokeer :raised_hands:

Great work on implementing the project requirements, however there are some issues to be solved before you get approved or move to the next project :muscle:
   
Changes required :writing_hand:  

   #### Please, follow the [design](https://www.behance.net/gallehttps://www.behance.net/gallery/29845175/CC-Global-Summit-2015ry/29845175/CC-Global-Summit-2015) given in the project requirements to implement the changes requested.

     * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 5 above)
Comment (CarlosE-Feliz) [reply index.html]: Sorry but what is this? 

Did you saw my video? my project works fine...

File: .hintrc (hunk 1 above)
Comment (CarlosE-Feliz) [reply .hintrc]: Yes, I copy and paste the linters of microverse!

File: index.html (hunk 3 above)
Comment (CarlosE-Feliz) [reply index.html]: I don't know what's happen with github pages but if you saw my [video](https://www.loom.com/share/0ea8113063bf40e4995f24516481e93a) you will see that my project works fine!

File: css/about.css [hunk 9]
```diff
@@ -126,101 +128,103 @@ hr {
...
+    background-color: #fff;
+    font-size: 22px;
+    display: flex;
+    height: 85px;
+  }
+
+  .nav-bar p {
+    justify-content: start;
+    margin: 12px 10px;
+  }
```
Comment (Lameck1) [inline css/about.css]: - [x] This section of the page was pointed out by the previous reviewer requesting that we at least match it as much as possible with the design. I think working on the spacings can help here. Our great undoing here is 
```CSS
justify-content: space-between;
```

When we use `space-between`, we spread the elements across the viewport when that is not the case with the original design. There is spacing to the right and to the left of this section.

# Comparison
--------------------------------------Original-----------------------------------------------
nav1

--------------------------------------Yours(After First Review)-----------------------------------------------
Capture


_Originally posted by @Emmyn5600 in https://github.com/xJokeer/Capstone_Microverse/pull/1#discussion_r682780124_

File: css/about.css [hunk 10]
```diff
@@ -126,101 +128,103 @@ hr {
 .d-none {
   display: none;
 }
+
 .menu_mobile {
   display: flex;
```
Comment (Lameck1) [inline css/about.css:132]: - [x] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @xJokeer,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 

 ## Required Changes ♻️

You've got this @xJokeer 💪🏼 , one of the major tasks being tested in this capstone is the ability to implement client requirements, and design is one of them. A client gives you a design because he/she desires the same look and feel on the final product. Let's strive to stick to the design as much as possible.

_Please check the comments under the review too._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .hintrc (hunk 1 above)
Comment (rloterh) [reply .hintrc]: This issue is not properly resolved 😿

File: js/about.js (hunk 2 above)
Comment (rloterh) [reply js/about.js]: This issue is well resolved! ✔️

File: index.html (hunk 3 above)
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

File: index.html (hunk 4 above)
Comment (rloterh) [reply index.html:49]: - This issue is not properly resolved 😿

File: .hintrc (hunk 1 above)
Comment (CarlosE-Feliz) [reply .hintrc]: So what do I have to do? I copied the template to setup the linters

File: index.html (hunk 5 above)
Comment (rloterh) [reply index.html]: This issue is not properly resolved 😿

File: .hintrc (hunk 1 above)
Comment (CarlosE-Feliz) [reply .hintrc]: image

File: index.html (hunk 6 above)
Comment (rloterh) [reply index.html]: This issue is not fully resolved 😿

File: css/about.css (hunk 7 above)
Comment (rloterh) [reply css/about.css]: This issue is not well resolved 😿

File: index.html (hunk 5 above)
Comment (CarlosE-Feliz) [reply index.html]: image 
Ammmm

File: css/about.css (hunk 9 above)
Comment (rloterh) [reply css/about.css]: This issue is not properly solved 😿

File: index.html (hunk 6 above)
Comment (CarlosE-Feliz) [reply index.html]: Is this necesary when I already get a second review and he approved those parts? can you please work with the second reviewer cause I don't have the time to play with reviewers. Why the second one approve these parts and you the tthird one come and say that they are not good

File: README.md [hunk 11]
```diff
@@ -16,12 +16,16 @@
 - HTML
 - CSS
 - Git
+- JavaSCript
+
```
Comment (rloterh) [inline README.md:20]: - [x] OPTIONAL
It's a nice and smart thing to give appropriate credit to **Cindy Shin**, the author of the original design, as required by the Creative Commons License. Kindly ensure to add this under the "Acknowledgements" section of your ReadMe

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Elim, 

You did great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [x] _Check the comments under the reviews of the previous code reviewers. Most of the requested changes by the previous code reviewer(s) above which I've commented as **not resolved**. Please take your time to improve upon the requested changes as the capstone project will mostly to be showcased on your portfolio, and therefore needs to look great! Optionally, kindly book an appointment with a full-time  TSE in case you may need extra support with this project_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] _Check the comments under below marked [OPTIONAL]_



Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .hintrc (hunk 1 above)
Comment (rloterh) [reply .hintrc]: Hi, here's is the link to the current `.hintrc`  https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc
Kindly check carefully and you'll notice that it's different from what you have

Comment (rloterh) [reply .hintrc]: Kindly check carefully and you'll notice that it's different from what you have

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

You're right, since the previous reviewer didn't ask for the other changes, they would be `[OPTIONAL]`

However, the navbar is still not looking like the given design (asked by the previous reviewer), once you fix that, you should get the project approved!

But I highly recommend fixing the other issues, as Robert said, this will be seen by recruiters and it would be better to be well made

Good luck && Keep rocking 🚀!

File: index.html (hunk 6 above)
Comment (rloterh) [reply index.html]: Hi, kindly note that the second reviewer simply highlighted on how you can improve a particular section. We're all working together ensure that you come out with a more solid project. Cheers! 🥰

**Comment by rloterh** [comment]
**Hello Elim!**

I've just realized I missed one very important point, by going as far back as the first review.

I'm sorry I revisited that part during my review. Kindly note, this was not on purpose 😊 

Happy coding! 🍷 💻 🚀 

Cheers!

File: README.md (hunk 11 above)
Comment (CarlosE-Feliz) [reply README.md:20]: Done, thx 😋

File: about.html [hunk 12]
```diff
@@ -38,7 +40,7 @@
             <section class="nav-bar d-flex d-none">
                 <p class="logo">Architects</p>
                 <ul class="nav-list d-flex">
-                    <li><a href="/index.html">Home</a></li>
+                    <li><a href="index.html">Home</a></li>
                     <li><a href="">Prizes</a></li>
                     <li><a href="">Biography</a></li>
                     <li><a href="">Projects</a></li>
```
Comment (Emmyn5600) [inline about.html:48]: - [x] Your page allows us to scroll x which doesn't look nice for the UX(user experience). Let us fix this by adding  overflow-x: hidden inside the body in our CSS. for more check this image.

main3

- [x] You can also check for the design to implement the top dark bar with the two icons, logout, English and My page as suggested.

File: css/about.css [hunk 13]
```diff
@@ -126,101 +127,109 @@ hr {
...
+  .section-social {
+    margin: 20px 8px 8px;
+    width: 20%;
+    justify-content: space-between;
+  }
+
+  .imgs {
+    flex-direction: row;
+  }
 }
```
Comment (Emmyn5600) [inline css/about.css]: - [ ] I recommend you to use the folowing code to make your background image covered in your body part, not to be duplicated.
`
  background-image: url(image);
  background-repeat: no-repeat;
  background-size: cover;`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @xJokeer 

Great work on implementing the changes requested by the previous TSE, however there are some issues to be solved before you get approved but you are almost there:muscle:
   
Changes required :writing_hand:  

     * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: about.html (hunk 12 above)
Comment (CarlosE-Feliz) [reply about.html:48]: Done, thx for the feedback!

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

You're right, those changes should be optional, the navbar looks fine to me now,

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @xJokeer  🔥 

Your capstone works as expected! Good work team👍🏽 

boomeasy

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**